src/OpenTelemetry/.publicApi/netstandard2.1/PublicAPI.Unshipped.txt

```diff
@@ -10,6 +10,24 @@ OpenTelemetry.Logs.LogRecord.TraceFlags.set -> void
 OpenTelemetry.Logs.LogRecord.TraceId.set -> void
 OpenTelemetry.Logs.LogRecord.TraceState.set -> void
 OpenTelemetry.Logs.OpenTelemetryLoggerOptions.ConfigureResource(System.Action<OpenTelemetry.Resources.ResourceBuilder!>! configure) -> OpenTelemetry.Logs.OpenTelemetryLoggerOptions!
+OpenTelemetry.SdkOptions
+OpenTelemetry.SdkOptions.AttributeCountLimit.get -> int?
+OpenTelemetry.SdkOptions.AttributeCountLimit.set -> void
+OpenTelemetry.SdkOptions.AttributeValueLengthLimit.get -> int?
+OpenTelemetry.SdkOptions.AttributeValueLengthLimit.set -> void
+OpenTelemetry.SdkOptions.SdkOptions() -> void
+OpenTelemetry.SdkOptions.SpanAttributeCountLimit.get -> int?
```

Comment:
I definitely envisioned `SdkOptions` becoming public and moving the SDK project. I'm still pondering if we'll ultimately want a different structure.

Some thoughts I've mulled over .. not sure how well if at all this plays with IOptions pattern.

```csharp
class AttributesLimits
{
    virtual int AttributeCountLimit;
    virtual int AttributeValueLengthLimit;
}

class SpanLimits : AttributeLimits
{
    override int AttributeCountLimit; // returns if set explicitly otherwise returns base value.
}

class TracingOptions
{
    SpanLimits SpanLimits;
}

class MetricOptions
{
    int MaxStreamLimit;
    int MaxPointsPerStreamLimit;

    PeriodicExportingMetricReaderOptions PeriodicExportingMetricReaderOptions;
}

class SdkOptions
{
    TracingOptions TracingOptons;
    MetricOptions MetricOptions;
}
```